testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -8182,6 +8182,19 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
         "CHAR(3) NOT NULL ARRAY NOT NULL");
     f1.checkScalar("map_keys(map('foo', 1, null, 2))", "[foo, null]",
         "CHAR(3) ARRAY NOT NULL");
+
+    f1.checkScalar("map_keys(map('foo', 1, null, 2))", "[foo, null]",
```

Comment:
I am assuming these have been checked against spark.
Perhaps you can add a comment, also listing the spark version - Spark has been known to change function behaviors in the past.

Reply:
done, I tested all versions after spark3.4